src/components/NcActionInput/NcActionInput.vue

```diff
@@ -406,7 +407,7 @@ $input-margin: 4px;
 
 	font-weight: normal;
 
-	&:deep(.material-design-icon) {
+	&:deep(.material-design-icon:not(.vs__open-indicator)) {
```

Comment:
Do these styles also incorrectly apply to the clear icon?

Reply:
Yes, they did. I fixed it.